hil/cli.py

```diff
@@ -247,17 +153,17 @@ def serve(port):
     # We need to import api here so that the functions within it get registered
     # (via `rest_call`), though we don't use it directly:
     # pylint: disable=unused-variable
-    from hil import api, rest
+    from hil import api
     server.init()
     migrations.check_db_schema()
     server.stop_orphan_consoles()
     rest.serve(port, debug=debug)
 
 
-@cmd
+@cli.command(name='serve_networks', help='Run the networking server')
```

Comment:
do we want to enforce a range of port numbers here as well?

Reply:
this process doesn't listen on a port. It's a simple python program reading from the database.